lib/src/DynamicProto.ts

```diff
@@ -431,9 +431,12 @@ function _checkPrototype(classProto:any, thisTarget:any) {
             visited.push(thisProto);
             thisProto = _getObjProto(thisProto);
         }
+
+        return false;
     }
 
-    return false;
+    // If objGetPrototypeOf doesn't exist then just assume everything is ok.
```

Comment:
This only affects "Old" runtimes IE8 and eariler.
So the side affect of this is that anyone using this will fail to start because the _checkPrototype always fails and throws.